packages/landing/static/index.css

```diff
@@ -38,7 +38,7 @@ button {
 }
 
 #map .ol-zoom-in {
-    border-radius: 4px 4px 0 0;
+    border-radius: 3 0 3;
```

Comment:
What's this? where are the units or `%`? why repeat the three but not the zero?

Reply:
ahh thats a bad copy pasta